### DISCUSSION TITLE
Improvements to searching the book for clipboard

### DISCUSSION THREAD

**Discussion by AceofSpades5757** [helix-editor/helix#10551] [General]
When looking for the different commands and keys on writing text to the clipboard(s), I had a hard time finding some of them due to the lack of the work `clipboard`, where `yank` is common. It'd be nice if the book's search function, or the documention amended, to allow searching for `clipboard` to include results for `yank`.

When searching through other help texts, I don't remember ever trying to use the work `yank` either.

If the settings could not be altered, then updating the wording may help. e.g. `Yank selection` to `Yank selection to clipboard`.

**Comment by the-mikedavis** [comment]
Yanking to the clipboard and yanking are different operations though. Yanking goes to a register and supports multiple selections while the clipboard only supports one possibly joined selection.

**Comment by AceofSpades5757** [comment]
@the-mikedavis Thank you for the clarification. That makes a lot of sense. Well, instead of adding anything additional to the documentation, it would be nice to make modifications to the search engine so that others might not run into the same issues as well.

**Comment by kirawi** [comment]
I'm not sure we can do anything about that without changing the files manually. It would probably need some changes upstream https://github.com/rust-lang/mdBook
